examples/conversational_retrieval_chain/server.py

```diff
@@ -87,12 +87,14 @@ class ChatHistory(BaseModel):
 
     chat_history: List[Tuple[str, str]] = Field(
         ...,
-        extra={"widget": {"type": "chat", "input": "question", "output": "answer"}},
+        extra={"widget": {"type": "chat", "input": "question", "output": "output"}},
```

Comment:
We should update this so that json path referencing is w/ respect to `output` not root